README-translated/README-Italian.md

```diff
@@ -82,18 +119,30 @@ L'ultima versione di Zenodo può essere menzionata come:
 
 > The Turing Way Community, & Scriberia. (2020, March 3). Illustrations from the Turing Way book dashes. Zenodo. http://doi.org/10.5281/zenodo.3695300
 
-Abbiamo utilizzato alcune di queste immagini nelle risposte del [Welcome Bot](https://github.com/apps/welcome) ai contributi dei nuovi membri nella nostra repository su GitHub.
+Abbiamo utilizzato alcune di queste immagini nelle risposte del [Welcome Bot](https://github.com/apps/welcome) ai contributi dei nuovi membri nel nostro repository su GitHub.
 
 ### Contattaci
 
-Abbiamo una chat room su gitter e ci farebbe piacere se passassi a dire ciao <https://gitter.im/alan-turing-institute/the-turing-way>.
+#### Email
+
+Puoi contattare il team *The Turing Way* scrivendo a [theturingway@gmail.com](mailto:theturingway@gmail.com).
+
+In alternativa, puoi contattare **Malvika Sharan** tramite [msharan@turing.ac.uk](mailto:msharan@turing.ac.uk) oppure **Kirstie Whitaker** scrivendo a [kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk).
```

Comment:
Maybe you could say to get in touch with them in English

Reply:
good point. I added it at a heading level as all emails/chats will be in English. Reckon it works this way?

Reply:
great!